flink-core/src/test/java/org/apache/flink/core/fs/SafetyNetCloseableRegistryTest.java

```diff
@@ -198,4 +211,27 @@ public void testReaperThreadSpawnAndStop() throws Exception {
 		}
 		Assert.assertFalse(SafetyNetCloseableRegistry.isReaperThreadRunning());
 	}
+
+	/**
+	 * Test for FLINK-17645.
```

Comment:
I think this line is not needed. It can be tracked via git log.

Reply:
I got it. I will delete this line.